presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveDynamicPartitionPruning.java

```diff
@@ -152,11 +153,12 @@ public void testJoinWithNonSelectiveBuildSide()
                 "SELECT * FROM partitioned_lineitem JOIN supplier ON partitioned_lineitem.suppkey = supplier.suppkey");
         assertEquals(result.getResult().getRowCount(), LINEITEM_COUNT);
 
+        /* Commented due to https://github.com/prestosql/presto/issues/5120
```

Comment:
Instead of commenting can we remove those lines ? We can add a TODO to point to the commit which has these changes ?

Reply:
done